### ISSUE TITLE
Failed to install some dependencies on Python 3.6

### ISSUE THREAD

**Issue by tarohi24** [megagonlabs/tagruler#1]
```
ERROR: Could not find a version that satisfies the requirement torch==1.1.0.post2
ERROR: No matching distribution found for torch==1.1.0.post2
```

**Comment by tarohi24** [comment]
I'll create a PR for it.

**Comment by tarohi24** [comment]
Fixed by 522a2b8a6cd54e632588173ce10d006533a48d84. Close it.
